erpnext/projects/doctype/task/task.json

```diff
@@ -382,14 +385,20 @@
    "fieldtype": "Date",
    "label": "Completed On",
    "mandatory_depends_on": "eval: doc.status == \"Completed\""
+  },
+  {
+   "fieldname": "template_task",
+   "fieldtype": "Data",
```

Comment:
@s-aga-r why did you use a "Data" field here, instead of a "Link", was this on purpose?